addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -340,10 +364,72 @@ const Wysiwyg = Widget.extend({
     redo: function () {
         this.odooEditor.historyRedo();
     },
+    /**
+     * Open the Alt dialog
+     * dialog give access to edit <img> alt and title
+     */
+    openAltDialog(params) {
+        let $baseNode = params.node ? params.node : this.lastTargetedElement;
+
+        let altDialog = new weWidgets.AltDialog(this, params, $baseNode);
+        altDialog.open();
+    },
+    /**
+     * Open the media dialog
+     * use to insert img / icons / videos
+     */
+    openMediaDialog(params) {
+        const range = Wysiwyg.getRange();
+        // we loose the current selection inside the content editable
+        // when we click on the media dialog button
+        // so we need to be able to restore the selection when the modal is closed
+        const restoreSelection = function() {
+            if(range.sc === null) return;
+            Wysiwyg.setRange(range.sc, range.so, range. ec, range.eo);
+        }
+
+        let $baseNode = params.node ? params.node : this.lastTargetedElement;
+        let mediaDialog = new weWidgets.MediaDialog(this, params, $baseNode);
+        mediaDialog.open();
+
+        mediaDialog.on('save', this, function(element) {
+            // restore saved html classes
+            if (params.htmlClass) element.className += " " + params.htmlClass;
+
+            if (this.lastTargetedElement.is('img') || params.node) {
+                $baseNode.replaceWith(element.outerHTML);
+            } else {
+                restoreSelection();
+                document.execCommand('insertHTML', false, element.outerHTML);
+            }
+        });
+        mediaDialog.on('closed', this,  function() {
+            // if the mediaDialog content has been saved
+            // the previous selection in not relevant anymore
+            if(mediaDialog.destroyAction !== 'save') restoreSelection();
+        });
+    },
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
 
+    _configureToolbar: function (options) {
+        console.log('------\nconfigure toolbar\n------')
+        let $toolbar = $(this.odooEditor.toolbar);
+        console.log($toolbar, options);
+        $toolbar.find('#media-modal').click((function(e) {
+            e.preventDefault();
+            e.stopImmediatePropagation();
+            e.stopPropagation();
+            this.openMediaDialog({});
+        }).bind(this));
```

Comment:
Why not replace this binding with an arrow function?
```js
$toolbar.find('#media-modal').click(e => {
    e.preventDefault();
    e.stopImmediatePropagation();
    e.stopPropagation();
    this.openMediaDialog({});
});
```

Reply:
yeah, indeed